tools/summaries/dc.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+
+import pandas as pd
+import requests
+
+_API_BASE_URL = "https://browser.datacommons.org/api/place/ranking"
+_RANKINGS_HEADER = "Rankings (in)"
```

Comment:
we could drop the parenthesis in the header. i wonder if that would make the csv easier to interpret by palm

Reply:
Done. Summaries seem to be similar with this change. In the larger context, these are the types of things we'll need to tinker with to come up with the prompt(s) that works best.